app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list: list) -> list:
+    persons = [Person(person_dict["name"], person_dict["age"])
+               for person_dict in people_list]
+
+    for person_dict in people_list:
+        person = Person.people[person_dict["name"]]
+        wife_name = person_dict.get("wife")
+        husband_name = person_dict.get("husband")
+
+        if wife_name and wife_name in Person.people:
```

Comment:
You could simplify this check to 
```suggestion
        if Person.people.get(wife_name):
```
Consider this approach in the future.